packages/compass-global-writes/src/components/index.tsx

```diff
@@ -98,7 +98,10 @@ function ShardingStateView({
 
 export function GlobalWrites({ shardingStatus }: GlobalWritesProps) {
   return (
-    <div className={containerStyles}>
+    <div
+      className={containerStyles}
+      data-testid={`${shardingStatus.toLowerCase()}-status`}
```

Comment:
I'd prefer something like `data-status={status}` and `data-testid=global-writes-container` - because status is more of a property of the workspace than an id (for example it could be generic `NOT_READY`). Alternatively the testid could be a combination of id and status

Reply:
`data-status={status}` reads better. i'll change it